webapp/src/libs/auth/AuthHelper.ts

```diff
@@ -9,13 +9,34 @@ import {
 } from '@azure/msal-browser';
 import debug from 'debug';
 import { Constants } from '../../Constants';
+import { ActiveUserInfo } from '../../redux/features/app/AppState';
+import { IChatUser } from '../models/ChatUser';
 import { TokenHelper } from './TokenHelper';
 
 const log = debug(Constants.debug.root).extend('authHelper');
 
+export const enum AuthType {
+    AAD = 'AzureAd',
+}
+
+export const DefaultChatUser: IChatUser = {
+    id: 'c05c61eb-65e4-4223-915a-fe72b0c9ece1',
```

Comment:
[from the backend](https://github.com/microsoft/chat-copilot/pull/126/files#diff-7b039d0ec1e1d963a9f72ca782023aa20a7b6ed4c471879eac6dc8d96a9b5b26R19). it's the same ID being used there

Reply:
Yup it's just a random GUID. It can be any GUID, just has to match on both sides.

Reply:
It might help to add a comment explaining that the default user is what is used when auth is set to None, and that the user id just needs to match what the backend expects.